ushahidi/platform-client
# Date picker fix

This pull request makes the following changes:
- Pauses the focus-trap when the date-picker is active
- Unpauses the focus-trap when the date-picker is closed
- Adjust the width to date/date-time fields when creating a post so all information is visible.

Testing checklist:
For filters
- Open the filter-dropdown
- Click on start-date or end-date
- [ ] Date-picker is opened
- [ ] It should be possible to select a date
- [ ] The filter-dropdown should remain open when the date-picker is open
- [ ] Repeat above for navigation with keyboard, tab to the start/end-date instead of clicking on it

For date-time field in post-creation
- Add a post for a survey that has a date/time field, select a date and time
- [ ] The whole selected date and time should be visible in the input box after selection.




Fixes ushahidi/platform#4190 .

Ping @ushahidi/platform
